Tools/autotest/quadplane.py

```diff
@@ -103,6 +103,40 @@ def get_disarm_delay(self):
     def set_autodisarm_delay(self, delay):
         self.set_parameter("LAND_DISARMDELAY", delay)
 
+    def test_motor_mask(self):
+        """Check operation of output_motor_mask"""
+        if not(int(self.get_parameter('Q_TILT_MASK')) & 1):
+#        or (int(self.get_parameter('Q_TAILSIT_MOTMX')) & 1)
+            self.progress("output_motor_mask not in use")
+            return
+        self.progress("Testing output_motor_mask")
+        self.reboot_sitl()
+        self.wait_ready_to_arm()
+
+        self.progress('Assert that SERVO5 is Motor1')
+        assert(33 == self.get_parameter('SERVO5_FUNCTION'))
+
+        self.progress("Testing MANUAL mode")
+        self.mavproxy.send('mode MANUAL\n')
+        self.wait_mode('MANUAL')
+        self.arm_vehicle()
+        self.progress("Raising throttle to 1800")
+        self.set_rc(3, 1800)
+        self.progress("Waiting for Motor1 to start")
+        self.wait_servo_channel_exceed_value(5, 1100)
+
+        self.reboot_sitl()
```

Comment:
Would be nice to avoid the reboot here.  Drop the servo, wait for it to reduce, then continue?

Reply:
"drop" a servo?

Reply:
Sorry, make the servo output channel return to its normal value :-)